pkg/apis/kops/componentconfig.go

```diff
@@ -655,8 +655,11 @@ type CloudConfiguration struct {
 	NodeInstancePrefix *string `json:"nodeInstancePrefix,omitempty"`
 
 	// vSphere cloud-config specs
+	VSphereUsername      *string `json:"vSphereUsername,omitempty"`
```

Comment:
These fields need to be added into pkg/apis/kops/v1alpha1/componentconfig.go and pkg/apis/kops/v1alpha2/componentconfig.go, and you also need to run conversion-gen to update the auto-generated files.